manifests/init.pp

```diff
@@ -53,4 +53,3 @@
     ],
   }
 }
-
```

Comment:
(My editor does too, but I disabled it and bound "clear trailing whitespace" to a key command instead)